opta/module_processors/aws_dns.py

```diff
@@ -176,7 +176,7 @@ def fetch_cert_body(self) -> Tuple[X509, str]:
                     "Certificate body can only have one certificate-- additional ones must go in the chain."
                 )
             try:
-                cert_obj = load_certificate(FILETYPE_PEM, cert_body)
+                cert_obj = load_certificate(FILETYPE_PEM, cert_body.encode("utf-8"))
```

Comment:
Intentional? Is this gonna have backwards compatibility issues?

Reply:
No it shouldn't-- when we updated the pipfile it updated a lot of things for better typing.